frontend/src/components/SignIn/SignIn.js

```diff
@@ -13,9 +13,19 @@ class SignIn extends Component {
       email: '',
       password: '',
       validationErrorText: '',
+      response: [],
+      suspendedUserList: [],
     };
   }
 
+  componentDidMount() {
+    getUsers().then((data) => {
```

Comment:
Once we actually have login working on the site, this page won't have access to the `getUsers` call, as you'll need to be an admin (otherwise we'd basically be publishing every user's email address publicly).